Components/Text/textutility.css

```diff
@@ -0,0 +1,48 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap");
+:root{
+    --primary-text-color:#3f3947;
+    --secondary-text-color:#696b79;

```

Comment:
try keeping the CSS variable names  more generic as you might need to use the same color as background.
Rest all is good!

Reply:
yes but this is only for text and there are variables defined in the component.css file for background & other colors.